aem/build.gradle.kts

```diff
@@ -61,6 +61,10 @@ aem {
             dependsOn(":develop")
         }
 
+        environmentUp {
```

Comment:
@mierzwid fixes `sh gradlew setup` when not separated instance & env setup with app build